sasdata/postprocess.py

```diff
@@ -14,3 +19,52 @@ def apply_fixes(data: SasData, mantid_unit_error=True):
         data = fix_mantid_units_error(data)
 
     return data
+
+
+def deduce_qz(data: SasData):
+    # if Qz is not already in the dataset
+    if 'Qz' not in data._data_contents:
+        # now check if we have Qx and Qy
+        if 'Qx' in data._data_contents and 'Qy' in data._data_contents:
```

Comment:
All three could be in a single check saving an indentation:

`if 'Qz' not in data._data_contents and 'Qx' in data._data_contents and 'Qy' in data._data_contents:`